src/sql/workbench/api/node/mainThreadNotebookDocumentsAndEditors.ts

```diff
@@ -361,7 +361,10 @@ export class MainThreadNotebookDocumentsAndEditors extends Disposable implements
 			pinned: !options.preview
 		};
 		let trusted = uri.scheme === Schemas.untitled;
-		let model = new NotebookInputModel(uri, undefined, trusted, undefined, undefined, undefined, options.connectionId);
+		// TODO fix to not instantiate the model. The Notebook input should only do that now?
```

Comment:
Can we fix this TODO in this PR? E.g. we should pass / set whatever's needed into the NotebookInput, and that should be the only thing that instantiates this model from now on.

Reply:
Fixed it with latest changes